### PR TITLE
Atmos adjacency subsystem revival

### PR DESCRIPTION
**Pull request by AnturK**
Revival of #40725

Same thing as previous PR just rebased and minor issue fixed.

### CODE REVIEW

**Comment by vuonojenmustaturska** [comment]
👍

**Comment by yoyobatty** [comment]
Nice

**Comment by yoyobatty** [comment]
Just curious, can you provide any numbers related to performance improvements or relief to tick drifting?

**Comment by AnturK** [comment]
I'll try to get something up but this is going to be be bit hard to measure outside of adjacency calls saved.

File: code/controllers/subsystem/adjacent_air.dm [hunk 1]
```diff
@@ -0,0 +1,35 @@
+SUBSYSTEM_DEF(adjacent_air)
+	name = "Atmos Adjacency"
+	flags = SS_BACKGROUND
+	runlevels = RUNLEVEL_GAME | RUNLEVEL_POSTGAME
+	wait = 10
```
Comment (JJRcop) [inline code/controllers/subsystem/adjacent_air.dm]: Any further discussion for this number?
https://github.com/tgstation/tgstation/pull/40725#discussion_r222858852

**Comment by optimumtact** [comment]
Thog not sure how this make difference

**Comment by AnturK** [comment]
Boom make turf change many time. Only need one. (Also applies to other stacked changeturfs)

**Comment by duncathan** [comment]
has this addressed the runtimes and other associated issues from the previous pr?

**Comment by AnturK** [comment]
The closed turf change yes, needs a testmerge before i can say if anything new cropped up.

**Comment by Arianya** [comment]
Currently testmerged on Bagil

**Comment by AnturK** [comment]
Thank

**Comment by Razharas** [comment]
I still dream of a pure mixture interconnection calculation instead of the turf/object proxy

**Comment by vuonojenmustaturska** [comment]
If you listen closely, you can hear the cicadas

**Comment by optimumtact** [comment]
where's the results of the test?

**Comment by AnturK** [comment]
It works fine, fills up nicely during carpet bombings. Can't think of objective way to measure performance gain in real round for this though.

**Pull request merged**